[2024-11-06T06:16:32.054+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_very_first_dag.my_second_task manual__2024-11-06T06:15:52.656757+00:00 [queued]>
[2024-11-06T06:16:32.078+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_very_first_dag.my_second_task manual__2024-11-06T06:15:52.656757+00:00 [queued]>
[2024-11-06T06:16:32.079+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-11-06T06:16:32.103+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): my_second_task> on 2024-11-06 06:15:52.656757+00:00
[2024-11-06T06:16:32.113+0000] {standard_task_runner.py:60} INFO - Started process 1277 to run task
[2024-11-06T06:16:32.118+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'my_very_first_dag', 'my_second_task', 'manual__2024-11-06T06:15:52.656757+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dag.py', '--cfg-path', '/tmp/tmpicjk8eqg']
[2024-11-06T06:16:32.122+0000] {standard_task_runner.py:88} INFO - Job 7: Subtask my_second_task
[2024-11-06T06:16:32.141+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-11-06T06:16:32.193+0000] {task_command.py:423} INFO - Running <TaskInstance: my_very_first_dag.my_second_task manual__2024-11-06T06:15:52.656757+00:00 [running]> on host 8b04ee1dc3cd
[2024-11-06T06:16:32.318+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_very_first_dag' AIRFLOW_CTX_TASK_ID='my_second_task' AIRFLOW_CTX_EXECUTION_DATE='2024-11-06T06:15:52.656757+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-06T06:15:52.656757+00:00'
[2024-11-06T06:16:32.320+0000] {logging_mixin.py:188} INFO - Hello from Airflow again
[2024-11-06T06:16:32.321+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-11-06T06:16:32.336+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=my_very_first_dag, task_id=my_second_task, execution_date=20241106T061552, start_date=20241106T061632, end_date=20241106T061632
[2024-11-06T06:16:32.371+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-11-06T06:16:32.398+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
